packages/renderer/src/lib/image/RunImage.svelte

```diff
@@ -69,68 +170,661 @@ async function startContainer() {
+<Route path="/*" let:meta>
+  {#if dataReady}
+    <NavPage
+      title="Create container from Image {imageDisplayName}"
+      searchEnabled="{false}"
+      subtitle="{image.tag}@{image.shortId} ">
+      <div slot="empty" class="bg-zinc-700 p-5 h-full">
+        <div class="bg-zinc-800 px-6 py-4 space-y-2 lg:px-8 sm:pb-6 xl:pb-8">
+          <section class="pf-c-page__main-tabs pf-m-limit-width">
+            <div class="pf-c-page__main-body">
+              <div class="pf-c-tabs pf-m-page-insets" id="open-tabs-example-tabs-list">
+                <ul class="pf-c-tabs__list">
+                  <li class="pf-c-tabs__item" class:pf-m-current="{meta.url === `/images/run/basic`}">
+                    <a
+                      href="/images/run/basic"
+                      class="pf-c-tabs__link"
+                      aria-controls="open-tabs-example-tabs-list-details-panel"
+                      id="open-tabs-example-tabs-list-details-link">
+                      <span class="pf-c-tabs__item-text">Basic</span>
+                    </a>
+                  </li>
+                  <li class="pf-c-tabs__item" class:pf-m-current="{meta.url === `/images/run/advanced`}">
+                    <a
+                      href="/images/run/advanced"
+                      class="pf-c-tabs__link"
+                      aria-controls="open-tabs-example-tabs-list-details-panel"
+                      id="open-tabs-example-tabs-list-details-link">
+                      <span class="pf-c-tabs__item-text">Advanced</span>
+                    </a>
+                  </li>
+                  <li class="pf-c-tabs__item" class:pf-m-current="{meta.url === `/images/run/networking`}">
+                    <a
+                      href="/images/run/networking"
+                      class="pf-c-tabs__link"
+                      aria-controls="open-tabs-example-tabs-list-yaml-panel"
+                      id="open-tabs-example-tabs-list-yaml-link">
+                      <span class="pf-c-tabs__item-text">Networking</span>
+                    </a>
+                  </li>
+                  <li class="pf-c-tabs__item" class:pf-m-current="{meta.url === `/images/run/security`}">
+                    <a
+                      href="/images/run/security"
+                      class="pf-c-tabs__link"
+                      aria-controls="open-tabs-example-tabs-list-yaml-panel"
+                      id="open-tabs-example-tabs-list-yaml-link">
+                      <span class="pf-c-tabs__item-text">Security</span>
+                    </a>
+                  </li>
+                </ul>
+              </div>
             </div>
-          {/each}
-        </div>
+          </section>
+          <div>
+            <Route path="/basic">
+              <div class="h-96 overflow-y-auto pr-4">
+                <label for="modalContainerName" class="block mb-2 text-sm font-medium text-gray-900 dark:text-gray-300"
+                  >Container Name:</label>
```

Comment:
```suggestion
                  >Container name:</label>
```

This is capitalized, but other parts (like "Environment variables") isn't. I think the form shouldn't have capitalization.